exporter/awsemfexporter/README.md

```diff
@@ -28,11 +28,20 @@ The following exporter configuration parameters are supported.
 | [`metric_declarations`](#metric_declaration) | List of rules for filtering exported metrics and their dimensions. |    [ ]   |
 
 ### <metric_declaration>
-A metric_declaration section characterizes a rule to be used to set dimensions for exported metrics, filtered by the incoming metrics' metric names.
+A metric_declaration section characterizes a rule to be used to set dimensions for exported metrics, filtered by the incoming metrics' labels and metric names.
 | Name              | Description                                                            | Default |
 | :---------------- | :--------------------------------------------------------------------- | ------- |
 | `dimensions`      | List of dimension sets to be exported.                                 |  [[ ]]   |
 | `metric_name_selectors` | List of regex strings to filter metric names by.                 |   [ ]    |
+| [`label_matchers`](#label_matcher)  | (Optional) list of label matching rules to filter metrics by their labels. This rule is applied to any metric that matches any of the label matchers. |   [ ]    |
+
+### <label_matcher>
```

Comment:
Should this section to be a subsection of  <metric_declaration> ?

Reply:
From our design discussion, the `label_matcher` is an additional attribute to the `metric_declaration`. So, yes, it is a subsection of `metric_declaration`. Should it not be?

Reply:
Sorry, should have pointed it out more directly, I was trying to say it should be `####` here instead of `###`

Reply:
Ah, got it! Fixed it in the latest commit :)